sdks/java/extensions/schemaio-expansion-service/build.gradle

```diff
@@ -58,6 +58,7 @@ dependencies {
     permitUnusedDeclared 'com.google.cloud:alloydb-jdbc-connector:1.2.0'
     testImplementation library.java.junit
     testImplementation library.java.mockito_core
+    runtimeOnly ("org.xerial:sqlite-jdbc:3.49.1.0")
```

Comment:
Should this be a `testImplementation`? Why do we need this?

Reply:
1. It contains the driver for connecting to the DB.
2. My understanding is that testImplementation is for tests inside that the schemaio-expansion-service module.  We need the runtimeOnly so that its packaged in the jar for use later.